.github/workflows/test-package.yml

```diff
@@ -39,17 +39,16 @@ jobs:
 
   test-linux:
     needs: analyze
-    # This job requires clang-10 which is the default on 20.04
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v2
       - uses: dart-lang/setup-dart@v1.0
         with:
           sdk: stable
       - name: Install dependencies
         run: dart pub get
-      - name: Install libclang-10-dev
-        run: sudo apt-get install libclang-10-dev
+      - name: Install libclang-14-dev
```

Comment:
README.md lists that we need at least clang 9.

We should not require a too new version, and we should run the CI on the oldest version we support.

Filed

* https://github.com/dart-lang/ffigen/issues/426

to track this.